demos/projects/PC/linux/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+# set global path
+set(BOARD_DEMO_CONFIG_PATH ${CMAKE_CURRENT_SOURCE_DIR}/config CACHE INTERNAL "Config path")
+set(BOARD_DEMO_FREERTOS_PORT_PATH ${FreeRTOS_Posix_PATH} CACHE INTERNAL "FreeRTOS Port used ")
+
+# include flags
+include(${CMAKE_CURRENT_SOURCE_DIR}/gcc_flags.cmake)
+
+# include config path as global
+include_directories(${BOARD_DEMO_CONFIG_PATH})
+
+# Add port specific source file
+target_sources(FreeRTOSPlus::TCPIP::PORT INTERFACE 
+    ${FreeRTOSPlus_PATH}/Source/FreeRTOS-Plus-TCP/portable/BufferManagement/BufferAllocation_2.c
+    ${FreeRTOSPlus_PATH}/Source/FreeRTOS-Plus-TCP/portable/NetworkInterface/linux/NetworkInterface.c
+    ../../../common/transport/sockets_wrapper_freertos_tcpip.c)
```

Comment:
is there a variable this can be relative from? like `${FreeRTOSPlus_PATH}`

Reply:
removed